src/directory-sync/interfaces/user.interface.ts

```diff
@@ -20,6 +20,7 @@ export interface User<
   }[];
   username: string;
   last_name: string;
+  job_title: string;
```

Comment:
This should be marked as nullable:

```suggestion
  job_title: string | null;
```

Reply:
👍 thanks! Made this change.